src/solvers.jl

```diff
@@ -168,6 +170,8 @@ function solve(m::Model; suppress_warnings=false,
         end
     end
 
+    relaxation && (m.internalModelLoaded = false)
```

Comment:
Maybe add a comment here like: ``# don't keep relaxed model in memory``? I had trouble reading this line on the first try.